web/packages/teleterm/src/ui/services/workspacesService/accessRequestsService.ts

```diff
@@ -76,7 +76,9 @@ export class AccessRequestsService {
     }
   }
 
-  async addOrRemoveResource(request: ResourceRequest): Promise<void> {
+  async addOrRemoveResource(
```

Comment:
Why not define `addOrRemoveResources` which accepts an array? Alternatively rename this function here to `addOrRemoveResources` and make it always accept an array instead.

I used to do this a lot in Ruby where a method would accept a thing or an array of things, but ultimately I think it's better to keep it simple.